spec/unit/filter_option_parser_spec.rb

```diff
@@ -125,8 +125,8 @@
         filter = subject.parse("inspect", "skip-files" => "@#{exclude_file}")
 
         expect(filter.to_array).to eq([
-          "/unmanaged_files/files/name=/foo",
-          "/unmanaged_files/files/name=/bar"
+          "/unmanaged_files/name=/foo",
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.